requirements.txt

```diff
@@ -1,5 +1,6 @@
 gitpython
 pyyaml
+conda ==4.1.12
 conda-build-all
 conda-build ==2.0.6
-requests !=2.12.0
+requests ==2.11.1 
```

Comment:
Interesting. So is requests still causing problems

Remember we had to pin requests on a few of our conda execute jobs